### ISSUE TITLE
Cannot add a location

### ISSUE THREAD

**Issue by jelemux** [nucleus-ffm/foss_warn#66]
**Expected behaviour:**
I don't know what exactly the intended behaviour is, but when I input a location, I expect some suggestions for locations or at least a save button.

**Actual behaviour:**
I don't get any suggestions or save button or whatever to save that location.
Screenshots: 
![Screenshot_20221106-210821_FOSS_Warn](https://user-images.githubusercontent.com/54023859/200192621-34a42c15-4382-49d7-a9a1-220066481410.png)
![Screenshot_20221106-210833_FOSS_Warn](https://user-images.githubusercontent.com/54023859/200192634-c9b2de4c-92c5-468d-80d3-013de79f4a89.png)

**Things I tried:**
Deleting the apps storage and cache, playing around with some settings.

**Soft-/Hardware information:**
App Version: 0.4.3
OS: /e/OS 1.4
Device: Fairphone 4

If any more information is needed, please say so and I'll try to add it.

**Comment by nucleus-ffm** [comment]
Are you sure you are connected to the Internet? You are connected to a VPN. If this is some kind of firewall like Netguard, have you checked if a connection is blocked for FOSS Warn?

**Comment by jelemux** [comment]
Yes, I was connected. The VPN is Blockada. I couldn't find anything in its logs that looked like it could cause this. 

I disabled it anyway and then adding a location worked fine, even after I reenabled it and deleted the storage of this app. 

So you were right it must have had something to do with Blockada but I don't know what exactly caused the issue.

Closing this now, if anyone else has this issue, we could reopen the issue and investigate it further.
